.eslintrc.js

```diff
@@ -341,6 +341,7 @@ module.exports = {
         'jsx-a11y/click-events-have-key-events': 'off',
         'jsx-a11y/anchor-has-content': 'off',
         'jsx-a11y/tabindex-no-positive': 'off',
+        'jsx-a11y/label-has-associated-control': 'off',
```

Comment:
Just curious, is there a plan to update `packages/kbn-ui-framework` and remove the exemption here, or are we leaving this alone (maybe because we plan to remove that package entirely)?

Reply:
@w33ble yep. From what I understand from Tim Roes - the plan is to remove kbn-ui-framework completely.